### PR TITLE
8295844: jdk/test/whitebox/CPUInfoTest.java failed with "not all features are known: expected true, was false"

### PR DESCRIPTION
**Pull request by vnkozlov**
[JDK-8295776](https://github.com/openjdk/jdk/commit/d50b6eb342e9ec96d1a01dafc317e00725dc84c0) added new x86 CPU flags checks but did not update  CPUInfoTest.java.
Add missing x86 CPU's flags to CPUInfoTest.java.

Tested with tier1 when CPUInfoTest.java is ran.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8295844](https://bugs.openjdk.org/browse/JDK-8295844): jdk/test/whitebox/CPUInfoTest.java failed with "not all features are known: expected true, was false"

### Reviewers
 * [Thomas Schatzl](https://openjdk.org/census#tschatzl) (@tschatzl - **Reviewer**) ⚠️ Review applies to [acd6b1f6](https://git.openjdk.org/jdk/pull/10837/files/acd6b1f649e67071025b161ee50113c1aa5b81df)
 * [Doug Simon](https://openjdk.org/census#dnsimon) (@dougxc - Committer) ⚠️ Review applies to [014e26ee](https://git.openjdk.org/jdk/pull/10837/files/014e26ee93ec20110091b8bae1296d16dfa63928)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/10837/head:pull/10837` \
`$ git checkout pull/10837`

Update a local copy of the PR: \
`$ git checkout pull/10837` \
`$ git pull https://git.openjdk.org/jdk pull/10837/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 10837`

View PR using the GUI difftool: \
`$ git pr show -t 10837`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/10837.diff">https://git.openjdk.org/jdk/pull/10837.diff</a>

</details>

### CODE REVIEW

**Review by tschatzl — APPROVED** [review]
Lgtm.

**Comment by vnkozlov** [comment]
There were 2 cases when we forgot update CPUInfoTest.java and JVMCI's AMD64.java. I added comment to vm_version_x86.hpp to remind about that. I also aligned flags in CPUInfoTest.java to match existing padding.

File: src/hotspot/cpu/x86/vm_version_x86.hpp [hunk 1]
```diff
@@ -311,6 +311,11 @@ class VM_Version : public Abstract_VM_Version {
   static address   _cpuinfo_segv_addr; // address of instruction which causes SEGV
   static address   _cpuinfo_cont_addr; // address of instruction after the one which causes SEGV
 
+  /*
+   * Update next files when declaring new flags:
```
Comment (dougxc) [inline src/hotspot/cpu/x86/vm_version_x86.hpp:315]: Update *following* files...

**Review by dougxc — APPROVED** [review]

File: src/hotspot/cpu/x86/vm_version_x86.hpp (hunk 1 above)
Comment (vnkozlov) [reply src/hotspot/cpu/x86/vm_version_x86.hpp]: done

**Pull request closed without merging**